### ISSUE TITLE
When using DataLocationProxy, HashLocationProxy's interval still runs

### ISSUE THREAD

**Issue by CodeOfficer** [quirkey/sammy#24]
DataLocationProxy should completely replace HashLocationProxy when used in an app

**Comment by jerodsanto** [comment]
I believe this is because Sammy() currently creates a new Sammy.Application sans an app_function argument so the HashLocationProxy is created and _startPolling is run. Only after this is the app_function run as a plugin which overwrites the location_proxy.

To fix this I modified [line 53](http://github.com/quirkey/sammy/blob/b60cfebef9508b938cb4a13c475d1c0acf47c257/lib/sammy.js#L53) to look like this:

```
app = Sammy.apps[selector] || new Sammy.Application(args.shift());
```

Tests still pass after the change, so I'm guessing this won't break any of the plugin stuff that follows.

**Comment by quirkey** [comment]
This was fixed in 0.6.
